Gifski/Gifski.swift

```diff
@@ -174,7 +177,7 @@ final class Gifski {
 					if result.isFinished {
 						do {
 							try gifski.finish()
-							completionHandlerOnce(.success(gifData as Data))
+							completionHandlerOnce(.success(gifData! as Data))
```

Comment:
Are you 100% sure the `!` is safe here?

Reply:
yep. we nil it `completionHandlerOnce` only.

Reply:
It would be more ideal if `context` was less `unsafe`. You would expect the variable to be available during the `writeCallback` lifecycle.

Reply:
> It would be more ideal if context was less unsafe. You would expect the variable to be available during the writeCallback lifecycle.

Yeah, I expected that too. // @kornelski

Reply:
@BowdusBrown Can you add a short comment there saying that `!` here is safe since we control when it's `nil`'d and it's `nil`'d afterwards. I prefer adding comments when I use force unwrapping, for readability.

Reply:
Will do